5common-units/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+	<head>
+        <meta charset="UTF-8" />
+		<title>CSS Units</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" />
+		<link href="https://fonts.googleapis.com/css2?family=Averia+Serif+Libre:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet" />
+		<link href="styles.css" rel="stylesheet" />
+	</head>
+	<body>
+        		<figure class="excerpt">
+			<h3 class="title">Advice from a Caterpillar</h3>
+			<blockquote class="quote">
+				<p class="quote-text">The Caterpillar was the first to speak.</p>
+				<p class="quote-text">“What <span class="large">size</span> do you want to be?” it asked.</p>
+				<p class="quote-text">“Oh, I'm not particular as to <span class="large">size</span>,” Alice hastily replied; “only one doesn't like changing so often, <span class="small">you know</span>.”</p>
+			</blockquote>
+			<figcaption class="citation">Excerpt from <cite>Alice's Adventures in Wonderland</cite> by Lewis Carroll.</figcaption>
+		</figure>
```

Comment:
Just a few extra tabs here :)

Reply:
I think I forgot to "format document." I need to get into that habit! :)